src/NSWallet/NSWallet.NetStandard/Views/Pages.cs

```diff
@@ -330,21 +330,33 @@ public static void ClosePage(INavigation navigation)
 
 		public static void Locker()
 		{
-            Application.Current.MainPage.Navigation.PushModalAsync(
-				new NavigationPage(new LockerPageView()), false);
+			// Hide screen content for iOS devices
+            if (Device.RuntimePlatform == Device.iOS) {
+                if (Application.Current.MainPage != null &&
+                    Application.Current.MainPage.Navigation != null) {
+                    Application.Current.MainPage.Navigation.PushModalAsync(
+                        new NavigationPage(new LockerPageView()), false);
+                }
+            }
 		}
 
-		public static void Close(bool isModal = false)
+		public static void Unlocker(bool isModal = false)
 		{
-            var navigationPage = Application.Current.MainPage.Navigation;
-
-            if (!isModal && navigationPage.NavigationStack.Count > 0) {
-                Application.Current.MainPage.Navigation.PopAsync(false);
+            // Unhide screen content for iOS devices
+            if (Device.RuntimePlatform == Device.iOS) {
```

Comment:
Check this on NSWallet.cs

Reply:
Let's leave it there for a while, we need better solution to secure the screen for iOS anyway, we will refactor this later, I will just put a comment there, so we do not forget